AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Support/Internal/StorageMultipartUploadClient.swift

```diff
@@ -135,6 +135,7 @@ class DefaultStorageMultipartUploadClient: StorageMultipartUploadClient {
                     let preSignedURL = try await serviceProxy.preSignedURLBuilder.getPreSignedURL(
                         key: self.key,
                         signingOperation: operation,
+                        metadata: nil,
```

Comment:
does this mean `metadata` won't be uploaded in case of multi part upload?

Reply:
I've updated multi upload to also include metadata for multipart uplaods.